tyk-headless/values.yaml

```diff
@@ -150,5 +154,9 @@ pump:
   tolerations: []
   affinity: {}
   extraEnvs: []
+  mounts: []
+    # - name: mount
+    #   filename: mount.txt
+    #   mountPath: /dev/null
```

Comment:
I think it's better to have a single generic example on how to mount files which can be used for all components. I don't think we need to do this for each component. 

Same thing for all other charts too.